rasa/core/channels/chat.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<body data-new-gr-c-s-check-loaded="14.1014.0" data-gr-ext-installed=""
+      data-new-gr-c-s-loaded="14.1014.0">
+<div class="logged-in-box auth0-box logged-in">
+    <h2>Happy chatting!</h2>
+    <div>You can include the chat on your own website using</div>
+    <div><code><pre>
+    &lt;div id="rasa-chat-widget" data-websocket-url="https://example.com/"&gt;&lt;/div&gt;
+&nbsp;&nbsp;&nbsp;&nbsp;&lt;script src="https://unpkg.com/@rasahq/rasa-chat" type="application/javascript"&gt;&lt;/script&gt;
+    </pre></code></div>
+</div>
+make sure to replace <code>https://example.com/</code> with the URL of your server
```

Comment:
Does this refer to the `data-websocket-url` in L9?

Reply:
yes it does